packages/node_modules/@ciscospark/plugin-conversation/src/conversation.js

```diff
@@ -469,10 +469,16 @@ const Conversation = SparkPlugin.extend({
   /**
    * Creates a ShareActivty for the specified conversation
    * @param {Object} conversation
+   * @param {Object} activity
    * @returns {ShareActivty}
    */
-  makeShare(conversation) {
-    return ShareActivity.create(conversation, null, this.spark);
+  makeShare(conversation, activity) {
```

Comment:
The idea behind `makeShare` is that the value it returns *is* the provisional activity. Why can't the web client use `makeShare` instead of a bare object?